force-app/main/default/objects/HOT_Request__c/fields/IsToHideFromHonor__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>IsToHideFromHonor__c</fullName>
+    <externalId>false</externalId>
+    <formula
+    >TODAY()&gt;= Date(Year(DateValue(StartTime__c) + 3), Month(DateValue(StartTime__c)+3), Day(DateValue(StartTime__c)))</formula>
```

Comment:
Her burde vi nok sjekke SeriesStartDate__c feltet. Vi har opplevd noen problemer med at StartTime ikke er helt up to date.

Reply:
SeriesEndDate__c